arch/M68K/M68KInstPrinter.c

```diff
@@ -244,10 +244,15 @@ void M68K_printInst(MCInst* MI, SStream* O, void* PrinterInfo)
 	detail = MI->flat_insn->detail;
 	if (detail) {
 		memcpy(&detail->m68k, ext, sizeof(cs_m68k));
+
+		memcpy(&detail->regs_read, &info->regs_read, info->regs_read_count * sizeof(uint16_t));
```

Comment:
need a bound check here of  info->regs_{read,write}_count size

nbRead = min(sizeof(detail->regs_read), info->regs_read_count * sizeof(uint16_t))
memcpy(&detail->regs_read, &info->regs_read, nbRead)
...